src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -2206,6 +2206,7 @@ metaitem.prospector.mode.ores=Ore Prospection Mode
 metaitem.prospector.mode.fluid=Fluid Prospection Mode
 metaitem.prospector.tooltip.ores=Scans Ores in a %s Chunk Radius
 metaitem.prospector.tooltip.fluids=Scans Ores and Fluids in a %s Chunk Radius
+behavior.prospector.not_enough_energy=Not Enough Energy!
```

Comment:
Why does this not have the `Analyzing Failed:` portion like the same key in zh_cn?

Reply:
That's exactly how it was before I removed the scanner lang keys without checking if anything else used them